### DISCUSSION TITLE
Promote Orchard Core as Headless CMS

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#14504] [General]
We should find some ways to let more people know than they can also use Orchard Core as a very powerful modular and extensible Headless CMS.

A documentation Guide could be useful. It would be interesting to find an appropriate static site generator to consume some exposed contents (I personnaly really like Astro).

We could reference it on the site https://jamstack.org/headless-cms/ by creating a simple PR adding a .md file of presentation. (Ex of another .NET CMS: https://github.com/jamstack/jamstack.org/blob/a1c88d82dacdf9c2c7745d15162d85c4d8075a4b/src/site/headless-cms/umbraco-heartcore.md).

If you have any suggestions to help promoting OC as Headless CMS, please advise.

**Comment by elaurentin** [comment]
I think having more plugins/modules/themes like we had before in OrchardCms will be a good start.
Modules like:
* Avatar
* Membership
* Event Management (Calendar, Map)
* eCommerce (store plugin with GraphQL or easy to support mobile App integration)
* Module(s) to integrate with Amazon/GDrive API
* Module(s) to support AI/ML back-end

Partnership/Sponsorship for those who want to support OC (monthly/yearly)

**Comment by agriffard** [comment]
PR https://github.com/jamstack/jamstack.org/pull/973 added.
Please tell me if the description is correct or can be improved.

I haven't joined any screenshots yet.

Preview: https://deploy-preview-973--jamstack-site.netlify.app/headless-cms/orchardcore/

**Reply by agriffard** [reply]
The repo seems to have a lot of PRs not merged for a long time: https://github.com/jamstack/jamstack.org/pulls

**Reply by Piedone** [reply]
Awesome!

**Reply by hishamco** [reply]
@agriffard please check my request changes https://github.com/jamstack/jamstack.org/pull/973#pullrequestreview-2318259658
